fronts-client/src/actions/__tests__/Cards.spec.ts

```diff
@@ -104,19 +104,15 @@ const insert = async (
     [uuid, id, undefined],
     collectionCapInfo ? collectionCapInfo.cap : Infinity
   );
+
+  const stateHere: any = getState();
```

Comment:
Casting this as `any` allows the tests to run without throwing lots of type errors. We don't need exact type matching in the tests.